src/extension/background-script/connectors/alby.ts

```diff
@@ -92,15 +94,16 @@ export default class Alby implements Connector {
       client.incomingInvoices({})
     )) as Invoice[];
 
-    const invoices: ConnectorInvoice[] = incomingInvoices.map(
-      (invoice, index): ConnectorInvoice => ({
+    const invoices: ConnectorTransaction[] = incomingInvoices.map(
```

Comment:
are both `getInvoices() and getTransactions()` still needed? shouldn't we have just one method?